azurerm/resource_arm_application_gateway.go

```diff
@@ -617,8 +626,10 @@ func resourceArmApplicationGateway() *schema.Resource {
 
 			// TODO: @tombuildsstuff deprecate this in favour of a full `ssl_protocol` block in the future
```

Comment:
since we've done this can we make this:

```suggestion
			// TODO: remove in 2.0
```